src/zenml/stack/flavor.py

```diff
@@ -34,6 +34,31 @@ def name(self) -> str:
             The flavor name.
         """
 
+    @property
+    def flavor_docs_url(self) -> str:
+        """A url to point at docs explaining this flavor.
+
+        Returns:
+            A flavor docs url.
+        """
+        from zenml import __version__
+
+        component_type = self.type.plural.replace("_", "-")
+        name = self.name.replace("_", "-")
+        base = f"https://docs.zenml.io/v/{__version__}"
+        url = f"{base}/component-gallery/{component_type}/{name}"
+
+        return url
+
+    @property
+    @abstractmethod
+    def logo_url(self) -> str:
```

Comment:
Maybe we could make this optional, seems like quite a hassle for users getting started writing a custom integration? The dashboard could just display some placeholder image I guess?

Reply:
Yeah, I intended this to be optional, will make sure it will be.